test/e2e/utils/recordset-utils.ts

```diff
@@ -192,6 +192,51 @@ export async function testRangeInputSubmitThenClear(page: Page, facet: Locator,
   await testClearAllFilters(page, facet, optionsCount - 1, pageSize);
 }
 
+/**
+ * used to test the input value that is set by chaise on load or after zoom/unzoom
+ * @param isFloat if the facet being tested is a float facet
+ * @param input the input to check the value of
+ * @param expectedVal the expected value of the input
+ */
+const testInputValue = async (isFloat: boolean, input: Locator, expectedVal: string) => {
+  let val = await input.getAttribute('value');
```

Comment:
If you knew the exact value or somehow could come up with a regular expression you could use `.toHaveValue`. But I don't think it's doable here. Right?

Reply:
Correct, we don't know the exact value because it behaves slightly different in github actions than it does locally. I am assuming that's because the float is calculated slightly differently in github actions compared to our local servers